api/accounts.go

```diff
@@ -214,6 +223,23 @@ func (a *API) accountSetHandler(msg *bearerstdapi.BearerStandardAPIdata, ctx *ht
 	return ctx.Send(data, bearerstdapi.HTTPstatusCodeOK)
 }
 
+// GET /accounts/treasurer
+// get the treasurer address
+func (a *API) treasurerHandler(msg *bearerstdapi.BearerStandardAPIdata, ctx *httprouter.HTTPContext) error {
+	acc, err := a.vocapp.State.Treasurer(true)
+	if err != nil || acc == nil {
```

Comment:
If `err != nil` does not mean that the treasurer does not exist but that there was some problem on the backend fetching it. Can we return a different error ? or is this done on purpose ?

Reply:
right